css/index.css

```diff
@@ -112,6 +114,46 @@ p {
     justify-content: center;
 }
 
+.teal{
```

Comment:
Using the !important property will ignore all other subsequent rules and apply the most !important rule. For this particular project, this is not needed. The best way to add 2 different rules to the same property would be to add 2 classes or a class and ID. All of the boxes can have the same class, but each box can have a different ID.